pkgs/by-name/pt/ptunnel/package.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchurl
+, libpcap
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ptunnel";
+  version = "0.72";
+
+  src = fetchurl {
+    url = "https://www.cs.uit.no/~daniels/PingTunnel/PingTunnel-${finalAttrs.version}.tar.gz";
+    hash = "sha256-sxj3qn2IkYtiadBUp+JvBPl9iHD0e9Sadsssmcc0B6Q=";
+  };
+
+  makeFlags = [
+    "prefix=${placeholder "out"}"
```

```suggestion
    "prefix=$(out)"
    "CC=cc"
```

minor nit: placeholder "out" is fine, but in makeFlags, can use make syntax and just set `prefix=$(out)`.

there is no `gcc` on default darwin stdenv but can just set `CC=cc` and that will find the c compiler for darwin and linux